application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -115,6 +150,81 @@ static boolean handleHasBotPermissions(@NotNull String actionVerb,
         return true;
     }
 
+    private static void handleAbsentTarget(@NotNull String actionVerb, @NotNull Interaction event) {
+        event
+            .reply("I can not %s the given user since they are not part of the guild anymore."
+                .formatted(actionVerb))
+            .setEphemeral(true)
+            .queue();
+    }
```

Comment:
Using an embed here would make it look more neater for the user

Reply:
It is not an embed. Did you fuck up your english and meant to say we should use an embed instead?

Would also have to be aligned with the other moderation commands where we are not using embeds for error messages. And with all other commands as well.

Id say, open a GH issue to discuss this for all commands (tex, tag, ... they all do it like that).